Source: William Orr <will@worrbase.com>
Upstream: https://github.com/nicolargo/glances/pull/245
Reason: Passes unit tests

---
 glances/glances.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/glances/glances.py b/glances/glances.py
index 131d179..bf64865 100644
--- a/glances/glances.py
+++ b/glances/glances.py
@@ -150,6 +150,9 @@ diskio_tag = True
 fs_tag = True
 process_tag = True
 
+# Global moved outside main for unit tests
+last_update_times = {}
+
 
 # Classes
 #========
-- 
1.8.2.1

